grammar.js

```diff
@@ -35,6 +35,54 @@ const numeric_types = [
 
 const primitive_types = numeric_types.concat(['bool', 'str', 'char'])
 
+const built_in_attributes = [
```

Comment:
Would it be possible to add a link to a section of the rust reference manual that lists those builtin attributes?

Reply:
Sure? I'm not sure how best to do that given this PR has already been merged - should I create a second PR for that and any other changes that might be requested on review?

The reference section is https://doc.rust-lang.org/reference/attributes.html#built-in-attributes-index.

Keeping this updated is a little more complicated than checking that list; according to the current reference, only "most" built-in attributes follow the meta item syntax (although I haven't yet found one that doesn't), and additionally there may be other well-known attributes (e.g. tool attributes) that do use the meta item syntax.

Reply:
You can make a second PR, or maybe because it's a small thing, you can fix it when you make another PR for fixing another problem.